HOWTO.dual.openssl

```diff
@@ -0,0 +1,49 @@
+############################################################################
+#
+# OpenSSL dual versions
+#
+# Works with CentOS 7
+#
+############################################################################
+#
+# Add in alternative OpenSSL support into /opt/openssl.  /opt/openssl is
+# chosen instead of the default of /usr/local so that existing utilities
+# continue to use the original version of OpenSSL and so only specific
+# applications that requires the newer version of OpenSSL will pick up the
+# new code.
+#
+# Download latest openssl.X.Y.Z.tar.gz from https://www.openssl.org/source/
```

Comment:
At the time of review, "latest" according to the cited resource would be `3.0.0-alpha4`. I think we should rather recommend, e. g. "the latest stable version".

Reply:
Updated as suggested